raiden-network/light-client
# Disables minting on mainnet

Fixes #1579  

**Short description**
Disable minting on mainnet and adjusts the text to remove minting from display when the network is mained

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Select mainnet in metamask
2. Go to try to connected a token
3. Go to select hub. The token should not have a mint button. UDC tooltip and dialog should only show info about deposit
4. UDC screen doesn't mention mint anywhere
